src/decoder/stream.rs

```diff
@@ -383,6 +383,18 @@ impl From<TextDecodingError> for DecodingError {
     }
 }
 
+/// Decode config
+#[derive(Default)]
+pub struct DecodeConfig {
+    ignore_text_chunk: bool,
+}
+
+impl DecodeConfig {
+    pub fn set_ignore_text_chunk(&mut self, ignore_text_chunk: bool) {
+        self.ignore_text_chunk = ignore_text_chunk;
+    }
+}
+
```

Comment:
I think the _user facing_ part was more consistent before. Then `Decoder` and `StreamingDecoder` have more similar interfaces.

```rust
pub(crate) struct DecodeConfig { … }

impl StreamingDecoder {
    pub fn set_ignore_text_chunk(&mut self, ignore_text_chunk: bool)  { … }
}
```